test/services/ParserTest.scala

```diff
@@ -64,7 +64,7 @@ class ParserTest
   object TestDataParser extends DataParser {
     val logger: Logger = Logger(getClass)
   }
-  object TestDataGenerator extends DataGenerator(mockAuditEvents, mockMetrics, mockParserUtil, realErsValidationConfigs, mockErsUtil, mockErsValidator)
+  object TestDataGenerator extends DataGenerator(mockAuditEvents, mockMetrics, mockParserUtil, realErsValidationConfigs, mockErsUtil, mockErsValidator, mockAppConfig)
```

Comment:
There are some tests on L110 -> L126 looking at CSOP sheet name mapping to the correct sheetType and sheetId, you may have already looked at this but if not might be worth adding some tests for CSOP V5 sheet names as well?

Reply:
sure, can add more